src/move_relearner.c

```diff
@@ -173,6 +175,7 @@ static EWRAM_DATA struct
     u8 moveListScrollArrowTask;                          /*0x113*/
     u8 moveDisplayArrowTask;                             /*0x114*/
     u16 scrollOffset;                                    /*0x116*/
+    u8 categoryIconSpriteId;                             //Physical/Special/Status category
```

Comment:
To be consistent the comment should refer to the actual offset rather than what it's used for (which the name conveys well enough)